DXMainClient/DXGUI/Multiplayer/GameFiltersPanel.cs

```diff
@@ -70,11 +70,13 @@ public override void Initialize()
                 gap, chkBoxHidePasswordedGames.Y + UIDesignConstants.BUTTON_HEIGHT + gap,
                 0, 0
             );
+            chkBoxHideIncompatibleGames.Visible = !ClientConfiguration.Instance.PreventIncompatibleGames;
```

Comment:
If the client def is preventing incompatible games, then this check box is not necessary.